src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -490,10 +492,21 @@ uct_dc_mlx5_iface_dci_detach(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
 static UCS_F_ALWAYS_INLINE void
 uct_dc_mlx5_iface_dci_schedule_release(uct_dc_mlx5_iface_t *iface, uint8_t dci)
 {
+    uint8_t pool_index;
+    uint8_t stack_top;
+
     ucs_assert(!uct_dc_mlx5_iface_is_dci_rand(iface));
     ucs_assert(dci != UCT_DC_MLX5_EP_NO_DCI);
 
-    UCS_BITMAP_SET(iface->tx.dci_release_bitmap, dci);
+    /* adding current DCI into release stack and mark pool for
+     * processing, see details in @ref uct_dc_mlx5_dci_pool_t description */
+    pool_index = uct_dc_mlx5_iface_dci_pool_index(iface, dci);
```

Comment:
init inline?

Reply:
> init inline?

may fail: in case if ```dci == UCT_DC_MLX5_EP_NO_DCI``` then it may crash

Reply:
uct_dc_mlx5_iface_dci_pool_index already has assert, can remove it from line 499

Reply:
> uct_dc_mlx5_iface_dci_pool_index already has assert, can remove it from line 499

ok, done